assets/js/favorites.js

```diff
@@ -0,0 +1,70 @@
+var getFavorites = function() {
+    var favoritesListEl = document.getElementById("favorites-list");
+    currentFavorites = localStorage.getItem('favorites');
+    currentFavorites = currentFavorites ? JSON.parse(currentFavorites) : []
```

Comment:
Really neat to see the use of the conditional (ternary) operator in our website!